workflowactions.md

```diff
@@ -28,4 +29,74 @@ Provide detailed information about a specific workflow action. An example JSON r
 
 The **options** property contains the list of actions the user is authorized to perform in this step:
 * The **edit_metadata** option implies the user can use the PATCH on the workflow item's submission sections to edit the metadata.
-* Other options are considered to be command options sent to REST using a [POST to the claimed task](claimedtasks.md#post-method-single-resource-level)
\ No newline at end of file
+* Other options are considered to be command options sent to REST using a [POST to the claimed task](claimedtasks.md#post-method-single-resource-level)
+
+The **advanced** property is `false` by default. When it's true, the action has more advanced behavior than simple buttons. Some samples are:
+* `edit_metadata` will redirect the user to the edit page
+* `assign_eperson` will show an EPerson lookup on a workflow task page
+* `rating` will show an option to enter a score on a workflow task page
+
+Although `edit_metadata` is also an advanced button with custom behavior, the current implementation deviates too much to fit in this framework.
```

Comment:
The 'Edit' button of the regular workflow's 2nd step (from option `edit_metada`) goes to a page very similar to the submission, at `{ui.url}/workflowitems/{id}/edit`, whilst our advanced options framework will navigate to a page which contains a simple metadata overview (non-editable) and the advanced option to be performed, at`/workflowitems/{workflowItemId}/advanced?workflow={advancedWorkflowActionName}&claimedTask={claimedTaskId}`.
Whilst it would be _possible_ to (re)create an advanced page for `edit_metadata`, there already was a page for this which is fairly user-friendly, only allowing reviewer to edit metadata that was accessible in submission. 
So `edit_metadata` is not configured as an advanced workflow step.